styles.css

```diff
@@ -0,0 +1,662 @@
+  background-size: cover;
+  background-position: 90% 100%;
+  color: #ec5242;
+  padding: 50px 24px;
+}
+
+.page-title {
+  /* -webkit-background-clip: text; */
+  background-clip: text;
+  background-image: url(https://w0.peakpx.com/wallpaper/194/474/HD-wallpaper-red-and-orange-texture-red-aesthetic.jpg);
+  color: transparent;
+  background-position: right;
+
+  /* font-size: 2.5rem; */
+  font-weight: 950;
+  font-size: 30px;
+}
+
+.home-text {
+  color: #272a31;
+  border: 3px solid #fff;
+  background-color: rgb(245, 245, 245);
+  padding: 12px;
+}
+
+.welcome-abt {
+  color: #ec5242 !important;
+}
+
+.date {
+  color: var(--black);
+  margin-top: 40px;
+}
+
+.hero {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  align-items: center;
+  background-color: var(--gray);
+  padding: 100px 0;
+}
+
+footer p {
+  color: #272a31;
+  font-size: 13px;
+}
+
+.past-header p {
+  font-size: 12px;
+  margin: 0 15px;
+}
+
+.hero p {
+  color: var(--black);
+  font-size: 14px;
+  font-weight: normal;
+}
+
+.contact {
+  text-decoration: underline;
+  font-weight: bold !important;
+}
+
+.past-weeks {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 15px;
+  margin-top: 25px;
+  margin-bottom: 30px;
+}
+
+.past-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  color: #272a31;
+  align-items: center;
+  text-align: center;
+  padding-top: 12px;
+}
+
+.event1 {
+  width: 60%;
+  height: 18vh;
+  contain: content;
+}
+
+.event1 img {
+  width: 100%;
+}
+
+.layer {
+  background: rgba(226, 0, 0, 0.6);
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  cursor: pointer;
+  color: #fff;
+  gap: 12px;
+  transition: 0.5s;
+}
+
+.layer:hover {
+  background: transparent;
+  color: transparent;
+}
+
+.program-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+}
+
+.prog-title {
+  display: flex;
+  flex-direction: column;
+
+  /* justify-content: center; */
+  align-items: center;
+}
+
+.program-title {
+  text-align: center;
+  padding-top: 35px;
+}
+
+.underline {
+  background-color: var(--red);
+  height: 1px;
+  width: 40px;
+  margin-top: 12px;
+  margin-bottom: 20px;
+}
+
+.underline1 {
+  background-color: var(--black);
+  height: 1px;
+  width: 20px;
+  margin-top: 8px;
+  margin-bottom: 8px;
+}
+
+.main-program {
+  width: 100vw;
+
+  /* height: 50vh; */
+  background-image: url(./images/main_program.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: #fff;
+}
+
+.programs {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  height: 100px;
+  width: 94%;
+  justify-content: space-between;
+
+  /* color: #fff; */
+  background-color: rgba(194, 181, 181, 0.5);
+
+  /* opacity: 0.1; */
```

Comment:
- [ ] [Optional] Kindly remove all commented code from your work in order to prevent your work from being messy and with this you'll be sure to have clean code.